handler_test.go

```diff
@@ -672,6 +672,72 @@ func TestHandler_AuthenticatedDatabases_UnauthorizedBasicAuth(t *testing.T) {
 	}
 }
 
```

Comment:
empty body will either blow up the json parser or return an empty struct and validation will fail.  But true, we should have a test for it.  I'll add one.